script/postrelease

```diff
@@ -23,7 +23,6 @@ exit(1) unless x.strip == 'yes'
 puts
 
 verbose('git fetch origin')
-verbose('git reset --hard origin/master')
```

Comment:
We run this from hotfix branch also. So doesn't make sense to do this.

Reply:
Okok, then we can remove the `puts "This will reset the HEAD to origin/master?"` line as well from [here](https://github.com/aws/amazon-chime-sdk-js/blob/master/script/postrelease#L16).